daffodil-core/src/main/resources/daffodil.sh

```diff
@@ -1,9 +0,0 @@
-#!/bin/bash
```

Comment:
Review tool won't let me add a comment on the prior file (index.html) because the diff is too large.

We should publish the HTML version of the spec on the apache daffodil web site for users. Right now the fact that this is published on our NCSA deliverables is the only place it is accessible to my knowledge.

Reply:
It's currently published on the apache website at http://daffodil.apache.org/docs/dfdl/ linked under the docs menu item.